CODEOWNERS

```diff
@@ -62,10 +62,14 @@ go.mod @fleetdm/go
 #
 # (see website/config/custom.js for DRIs of other paths not listed here)
 ##############################################################################################
-/docs                                           @rachaelshaw
+/docs                                           @mikermcneil
 /docs/Using-Fleet/REST-API.md                   @rachaelshaw # « REST API reference documentation
 /docs/Contributing/API-for-contributors.md      @rachaelshaw # « Advanced / contributors-only API reference documentation
 /schema                                         @rachaelshaw # « Data tables (osquery/fleetd schema) documentation
+/docs/01-Using-Fleet/standard-query-library/standard-query-library.yml @rachaelshaw # « Device health checks (https://fleetdm.com/queries)
+/docs/Configuration/agent-configuration.md         @rachaelshaw # « Configuration surface: Agent
+/docs/Configuration/fleet-server-configuration.md  @rachaelshaw # « Configuration surface: Fleet server
+/docs/Configuration/configuration-files/README.md  @rachaelshaw # « Configuration surface: Fleet server (even more things)  FUTURE: consolidate this.
```

Comment:
@mike-j-thomas yo this is the kind of thing I was talking about that's gonna have to change when you and eric change that other thing.  No action needed but wanted to make sure you had context.